src/styles/container.scss

```diff
@@ -0,0 +1,23 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  padding: 50px 40px;
+  justify-content: space-between;
+  gap: 46px;
+
+  @media (min-width: 1200px) {
+    padding-inline: 128px;
+  }
```

Comment:
The `padding-inline` property is not widely supported yet. It's better to use `padding-left` and `padding-right` for better browser compatibility.